docs/gpa.md

```diff
@@ -194,28 +194,26 @@ The information included in the notice of intended procurement covers all the in
 
 </td>
       </tr>
-<!--
       <tr>
         <td>VII:4</td>
         <td>Procuring entities are encouraged to publish in the appropriate paper or electronic medium listed in Appendix III as early as possible in each fiscal year a notice regarding their future procurement plans (hereinafter referred to as "notice of planned procurement"). The notice of planned procurement should include the subject-matter of the procurement and the planned date of the publication of the notice of intended procurement.</td>
         <td markdown=1>
 
-* 
+* Enter *the subject-matter of the procurement* in `planning/rationale`
```

Comment:
I don't think this fits the semantics of `planning/rationale`. I think we should instead map it to `tender/description`. If a later release updates that description with more specific information, that's fine, and what is expected.

Reply:
Done